### DISCUSSION TITLE
Type error inside i18n.ts and navigation.ts

### DISCUSSION THREAD

**Discussion by thebossjon** [amannn/next-intl#1050] [Q&A]
Hi,
I can't figure out what the error is. any ideas? I am using tureborepo. next 14.2.1

i18n.ts
![image](https://github.com/amannn/next-intl/assets/168740871/66361423-368b-4eb0-8b40-e59ac8358c3d)
navigation.ts
![image](https://github.com/amannn/next-intl/assets/168740871/6487d576-aecf-4899-9345-f104bcefcaf8)

**Comment by amannn** [comment]
This is currently being discussed in https://github.com/amannn/next-intl/discussions/1014
